Tasks/Common/webdeployment-common/deployusingmsdeploy.ts

```diff
@@ -93,7 +142,15 @@ async function executeMSDeploy(msDeployCmdArgs) {
     });
 
     try {
-        await tl.exec("msdeploy", msDeployCmdArgs, <any>{failOnStdErr: true, errStream: errObj});
+        tl.debug("the argument string is:");
+        tl.debug(msDeployCmdArgs);
```

Comment:
@bryanmacfarlane , Rephrasing the sentence.

We register the secrets with agent using the below line: (Which I called as **masking**)
`console.log("##vso[task.setvariable variable=websiteUserName;issecret=true;]" + A);`

I thought it would be better to register the secrets closer to the place, where they are used and suggested it.